.github/workflows/tests_linux.yml

```diff
@@ -30,13 +30,15 @@ jobs:
           python-version: '3.7'
 
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2 # for codecov
```

Comment:
Nice -- wasn't aware this is an option.

Reply:
Yes, Codecov raises this warning. I think this is the reason why CodeCov sometimes couldn't merge coverage data uploaded from the same commit has.